app/controllers/api/v1/holidays_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V1::HolidaysController < Api::V1::BaseController
+  skip_before_action :doorkeeper_authorize!, only: :index
+  expose :holiday,  -> { Holiday.find(params[:id]) }
+  expose :holidays, -> { Holiday.all }
+
+
+  def index
+    render_api(holidays, :ok, each_serializer: HolidaySerializer)
+  end
+
+  def create
+    created_holiday = holidays.create(holiday_params)
+    render_api(created_holiday.id, :created)
+  end
+
+  def update
+    holiday.update(holiday_params)
```

Comment:
what about validation?

Reply:
What kind of validation should be added? I think it`s not necessary

Reply:
you have a validation rules [here] (https://github.com/milestep/ActiveMile/pull/121/files#diff-b26a0363e77ca3d219b6048e58a31b2cR1)
so what if I enter blank name and click update?
the back-end doesn't check if the record was successfully updated, in all cases it returns `render_api(:accepted)`

Reply:
validation added